src/components/ui/span.stories.tsx

```
@@ -0,0 +1,185 @@
+    children: "✓ Verified",
+  },
+};
+
+export const Warning: Story = {
+  args: {
+    variant: "warning",
+    children: "⚠ Pending",
+  },
+};
+
+export const Error: Story = {
```

Biome (2.3.13)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)